spec/unit/puppet_x/sql_connection_spec.rb

```diff
@@ -114,20 +119,22 @@ def stub_connection
     end
     context 'return result with errors' do
       it {
-        subject.stubs(:win32_exception).returns(Exception)
+        stub_connection
+        @connection.Errors.stubs(:count).returns(1)
+        @connection.Errors.stubs(:Description).returns("SQL Error in Connection")
```

Comment:
Maybe mocking library makes this magically work... but I read this as supplying a value for:

```ruby
@connection.Errors.Description
```

When I think we're interested in:

```ruby
@connection.Errors(0).Description
```